src/components/content/order/serviceActions/ServiceActionSubmitResultDetails.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
+import { Typography } from 'antd';
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import submitResultStyles from '../../../../styles/submit-result.module.css';
+
+function ServiceActionSubmitResultDetails({
```

Comment:
is this exactly same as service configuration change? It's better we change it to a common place - ServiceChangeResultDetails